lib/postgres.go

```diff
@@ -2916,3 +2944,50 @@ func (postgres *Postgres) GetNotifications(publicKey string) ([]*PGNotification,
 
 	return notifications, nil
 }
+
+//
+// Postgres Test tooling
+//
+
+// Drop all tables in the postgres Db.
+func (postgres *Postgres) resetDatabase() error {
```

Comment:
We can get this functionality for free with embedded postgres. There's a lot of logic here that assumes a local docker postgres for easier testing. IMO no reason to spin our own solution.

Reply:
agreed with MF^